CHANGELOG.rst

```diff
@@ -1,6 +1,109 @@
+- [Runs][Python] Expose ``RunData`` fields (``metrics``, ``params``, ``tags``) as dictionaries (#1078, @smurching)
+- [Runs] ``RunInfo.status``'s type is now string (#1264, @mparkhe)
+- [Installation] The MLflow python package no longer depends on ``scikit-learn``, ``mleap``, or ``boto3``. Users who want to use the ``scikit-learn`` support, the ``MLeap`` support, or ``s3`` artifact repository / ``sagemaker`` support will have to install these respective dependencies explicitly. (#1223, @aarondav)
+
+More features and improvements:
+
+- [Tracking][DB] Non-default driver support for SQLAlchemy backends: ``db+driver`` is now a valid tracking backend URI scheme (#1297, @drewmcdonald)
+- [Tracking] Validate backend store URI before starting tracking server (#1218, @luke-zhu, @sueann)
+- Add ``view_type`` argument to ``MlflowClient.list_experiments()`` in Python. (#1212, @smurching)
+- [Python] Dictionary values provided to ``mlflow.log_params`` and ``mlflow.set_tags`` can now be non-string types (e.g., numbers), and they will be automatically converted to strings. (#1364, @aarondav)
+- Add ``GetMetricHistory`` client API in Python and Java corresponding to the REST API. (#1178, @smurching)
+- [Tracking][R] API additions to be at parity with REST API and Python (#1122, @kevinykuo)
+- [Artifacts] Hadoop artifact repository with Kerberos authorization support  (#1011, @jaroslawk)
+- [Artifacts] Don't copy local artifacts in ``ArtifactRepository.download_artifacts`` to avoid having many copies of large model files in serving (#1307, @andrewmchen)
+- Support GCS in download utilities. ``gs://bucket/path`` files are now supported by the ``mlflow download`` CLI command and as parameters of type ``path`` in MLProject files. (#1168, @drewmcdonald)
+- [Python][Models] All python models exported by MLflow now declare ``mlflow`` as a dependency by default. In addition, we introduce a flag ``--install-mlflow`` users can pass to ``mlflow models serve`` and ``mlflow models predict`` methods to force installation of the latest version of MLflow into the model's environment. (#1308, @tomasatdatabricks)
+- [Python][Models] Update model flavors to lazily import dependencies. Modules that define Model flavors now import extra dependencies such as ``tensorflow``, ``scikit-learn``, and ``pytorch`` inside individual _methods_, ensuring that these modules can be imported and explored even if the dependencies have not been installed on a user's system. Also, the ``DEFAULT_CONDA_ENVIRONMENT`` module variable has been replaced with a ``get_default_conda_env()`` function for each flavor.
+- [Serving] Switch to ``gunicorn`` for serving of python models for better performance. This does not change the user interface. (#1322, @tomasatdatabricks)
+- [SageMaker][Deployment] Use the uniquely-generated model name as the S3 bucket prefix instead of requiring one from user. (#1183, @dbczumar)
```

Comment:
Use the uniquely-generated model name as the S3 bucket prefix instead of requiring one from user.  -> Use the uniquely-generated model name as the S3 bucket prefix instead of requiring one.